Libraries/CommonFunctions.psm1

```diff
@@ -633,6 +633,12 @@ function Install-CustomKernel ($CustomKernel, $allVMData, [switch]$RestartAfterU
 				}
 				return $true
 			} else {
+                $customKernelAlreadyInstall="CUSTOM_KERNEL_ALREADY_INSTALLED"
+                $currentStatus = Run-LinuxCmd -ip $job.PublicIP -port $job.SSHPort -username $user -password $password -command "tail -n 1 build-CustomKernel.txt"
```

Comment:
this $job is out of the scope line 580 foreach ( $job in $packageInstallJobs ) {